src/script/conversation/ConversationRepository.ts

```diff
@@ -3289,7 +3277,7 @@ export class ConversationRepository {
       await this.clearConversationContent(conversationEntity, conversationEntity.cleared_timestamp());
     }
 
-    if (isActiveConversation && (conversationEntity.is_archived() || conversationEntity.is_cleared())) {
+    if (isActiveConversation && conversationEntity.is_archived()) {
```

Comment:
We still allow user to see cleared conversation, we do not redirect them to next conversation.